components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/policy/PolicyConstants.java

```diff
@@ -120,4 +120,8 @@ public class PolicyConstants {
     
     public static final String  THROTTLING_TIER_CONTENT_AWARE_SEPERATOR = ">";
 
+    public static final String EXPORT_POLICY_TYPE_YAML ="YAML";
```

Comment:
Let's move these to [1].

[1] https://github.com/wso2/carbon-apimgt/blob/master/components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/importexport/ImportExportConstants.java

Reply:
And there is a formatting issue in this line

Reply:
Fixed with #30d0ecb]